packages/eslint-config-mavenlint-react/package.json

```diff
@@ -6,12 +6,12 @@
   "repository": "https://github.com/mavenlink/mavenlint.git",
   "license": "MIT",
   "peerDependencies": {
-    "eslint": ">=4.14.0",
-    "eslint-plugin-jsx-a11y": ">= 5.1.1",
-    "eslint-plugin-react": ">=7.5.1"
+    "eslint": ">=4.14.0"
   },
   "dependencies": {
     "eslint-config-mavenlint": "^3.1.2",
-    "eslint-plugin-mavenlint": "^0.4.1"
+    "eslint-plugin-jsx-a11y": "^5.1.1",
+    "eslint-plugin-mavenlint": "^0.4.1",
+    "eslint-plugin-react": "^7.11.1"
```

Comment:
The main point of this PR.

Note that jsx-a11y and react plugins have moved to the "dependencies" section, and I'll make a PR to not explicitly have a dependency on these in big Maven.

The benefit I think to this is that now I don't have to manage these versions across the couple of projects that use mavenlint.